sql/moz-fx-data-shared-prod/subscription_platform_derived/active_subscriptions_v1/query.sql

```diff
@@ -1,5 +1,5 @@
-{#- format off #}
 WITH product_union AS (
+{#- format off #}
```

Comment:
This isn't strictly necessary, but it served as a good test of the fix to Jinja comment formatting, and I think the only reason the comment wasn't put here in the first place is `bqetl format` was misformatting it to combine it with the preivous line, like
```sql
WITH product_union AS ({#- format off #}
```